stock_no_negative/views/product.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  © 2015-2016 Akretion (http://www.akretion.com/)
+  @author Alexis de Lattre <alexis.delattre@akretion.com>
+  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+-->
+
+<odoo>
+
+    <record id="view_template_property_form" model="ir.ui.view">
+        <field name="name">stock_no_negative.product.template.form</field>
+        <field name="model">product.template</field>
+        <field name="inherit_id" ref="stock.view_template_property_form" />
+        <field name="arch" type="xml">
+            <field name="categ_id" position="after">
+                <field name="allow_negative_stock"
```

Comment:
I think that this field should only be accessible to a specific security group. You basically prevent negative stocks, but allow any user to override this rule just going to the product and changing it. What do you think?